src/vs/workbench/contrib/chat/browser/chatAttachmentModel.ts

```diff
@@ -57,6 +58,15 @@ export class ChatAttachmentModel extends Disposable {
 		this._onDidChangeContext.fire();
 	}
 
+	async attachScreenshot(): Promise<void> {
```

Comment:
I don't think the ChatAttachmentModel should do this, something else can take a screenshot and attach it

Reply:
https://github.com/microsoft/vscode/blob/e87374a2791f389e71ba7b3d04a69c6b74ecf728/src/vs/workbench/contrib/chat/browser/chatAttachmentModel.ts#L41

addFile does the same thing

Reply:
I can rename to align - `addScreenshot`

Reply:
hm, I guess I see your point. that the call to create screenshot shouldn't be here